### PR TITLE
fix(postman): throw exception for undeclared variable

### LINKED ISSUE #70: Introduce the dry-run mode for converters
We have to introduce the dry-run mode for converters to be able to perform some sanity checks:
1. Variables are defined
2. URLs are valid according to WHATWG
3. Request has a body according to HTTP/1.1

### PR DESCRIPTION
**Pull request by pmstss**
fixes #70

### CODE REVIEW

File: packages/postman/src/parser/BaseVariableParser.ts [hunk 1]
```diff
@@ -34,4 +36,8 @@ export abstract class BaseVariableParser implements VariableParser {
         return this.generators.randomAlphaNumeric();
     }
   }
+
+  protected isDryRun(): boolean {
```
Comment (derevnjuk) [inline packages/postman/src/parser/BaseVariableParser.ts:40]: What about using getter instead?

**Review by derevnjuk — APPROVED** [review]

**Review by derevnjuk — COMMENTED** [review]
@pmstss you know, looking at converter I realized that we don't need `dryRun` option. In both modes, it should raise an error that can help me to indicate whether the file is "convertible" or not.

File: packages/postman/src/parser/EnvVariableParser.ts [hunk 2]
```diff
@@ -21,25 +21,26 @@ export class EnvVariableParser extends BaseVariableParser {
...
     }
 
-    if (!variable || !variable.value) {
-      return match;
+    if (!variable) {
+      throw new Error(`Undefined variable: \`${token}\``);
     }
 
-    return variable.value;
+    return variable.value ?? match;
```
Comment (derevnjuk) [inline packages/postman/src/parser/EnvVariableParser.ts:44]: @pmstss Should we perceive 0 and empty strings as valid variable values?

Comment (pmstss) [reply packages/postman/src/parser/EnvVariableParser.ts:44]: yes, as discussed - w/o additional contextual information it's not possible to make a decision: e. g. empty value of variable inside hostname perhaps should be an error, but empty value for optional search parameter like `query` could be the desired

**Review by pmstss — COMMENTED** [review]
approved

**Pull request merged**